content/en/containers/kubernetes/distributions.md

```diff
@@ -330,7 +313,22 @@ agents:
 {{% /tab %}}
 {{< /tabs >}}
 
-**Note**: Network Performance Monitoring is not supported for GKE Autopilot.
+Similarly when using [GKE Autopilot Compute classes][11] to run workloads that have specific hardware requirements, take note of the [Taints][9] that GKE Autopilot is applying to these specific nodes and add matching Tolerations to the Agent DaemonSet. You can match the Tolerations on your corresponding pods. For example for the `Scale-Out` compute class use a Toleration like:
```

Comment:
```suggestion
Similarly when using [GKE Autopilot Compute classes][11] to run workloads that have specific hardware requirements, take note of the [Taints][9] that GKE Autopilot is applying to these specific nodes and add matching tolerations to the Agent DaemonSet. You can match the tolerations on your corresponding pods. For example for the `Scale-Out` compute class use a toleration like:
```

Reply:
Is this generally industry accepted as capitalized? If not should probably use lower-case

Reply:
Good call out, I thought K8s did so I mirrored, but I must have skimmed to quickly. Removed those. 
https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/